TetrisDotnet/Code/UI/Elements/HeldPieceUI.cs

```diff
@@ -0,0 +1,114 @@
+using SFML.Graphics;
+using SFML.System;
+
+namespace TetrisDotnet.Code.UI.Elements
+{
+	// ReSharper disable once InconsistentNaming
+	public class HeldPieceUI
+	{
+		Sprite[,] holdSprite4x4;
+		Sprite[,] holdSprite1x4;
+		Sprite[,] holdSprite3x3;
```

Comment:
In theory the PieceQueue will have the same shit

Reply:
oh yeah definitely. This code is just transferred over. Hasn't been modified at all. needs to be redone. Already a task for this. It's entirely possible that this class will be completely removed once we bring back the held piece visual functionality.